pkg/resources/csi/vmwareclouddirector/deployment.go

```diff
@@ -0,0 +1,269 @@
+				{
+					Name:            "csi-attacher",
+					ImagePullPolicy: corev1.PullIfNotPresent,
+					Image:           registry.Must(data.RewriteImage("registry.k8s.io/sig-storage/csi-attacher:v4.3.0")),
+					Args: []string{
+						"--csi-address=$(ADDRESS)",
+						"--kubeconfig=/etc/kubernetes/kubeconfig/kubeconfig",
+						"--timeout=180s",
+						"--v=5",
+					},
+					Env: []corev1.EnvVar{
+						{
+							Name:  "ADDRESS",
+							Value: "unix:///var/lib/csi/sockets/pluginproxy/csi.sock",
+						},
+					},
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "socket-dir",
+							MountPath: "/var/lib/csi/sockets/pluginproxy/",
+						},
+						{
+							Name:      resources.VMwareCloudDirectorCSIKubeconfigSecretName,
+							MountPath: "/etc/kubernetes/kubeconfig",
+							ReadOnly:  true,
+						},
+					},
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("10m"),
+							corev1.ResourceMemory: resource.MustParse("24Mi"),
+						},
+						Limits: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("100m"),
+							corev1.ResourceMemory: resource.MustParse("64Mi"),
+						},
+					},
+				},
+				{
+					Name:            "csi-provisioner",
+					ImagePullPolicy: corev1.PullIfNotPresent,
+					Image:           registry.Must(data.RewriteImage("registry.k8s.io/sig-storage/csi-provisioner:v2.2.2")),
+					Args: []string{
+						"--csi-address=$(ADDRESS)",
+						"--kubeconfig=/etc/kubernetes/kubeconfig/kubeconfig",
+						"--default-fstype=ext4",
+						"--timeout=300s",
+						"--v=5",
+					},
+					Env: []corev1.EnvVar{
+						{
+							Name:  "ADDRESS",
+							Value: "unix:///var/lib/csi/sockets/pluginproxy/csi.sock",
+						},
+					},
+					VolumeMounts: []corev1.VolumeMount{
+						{
+							Name:      "socket-dir",
+							MountPath: "/var/lib/csi/sockets/pluginproxy/",
+						},
+						{
+							Name:      resources.VMwareCloudDirectorCSIKubeconfigSecretName,
+							MountPath: "/etc/kubernetes/kubeconfig",
+							ReadOnly:  true,
+						},
+					},
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("10m"),
+							corev1.ResourceMemory: resource.MustParse("24Mi"),
+						},
+						Limits: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse("100m"),
+							corev1.ResourceMemory: resource.MustParse("64Mi"),
+						},
+					},
+				},
+				{
+					Name:            "vcd-csi-plugin",
+					ImagePullPolicy: corev1.PullIfNotPresent,
+					Image:           registry.Must(data.RewriteImage("projects.registry.vmware.com/vmware-cloud-director/cloud-director-named-disk-csi-driver:" + csiVersion)),
+					Command:         []string{"/opt/vcloud/bin/cloud-director-named-disk-csi-driver"},
+					Args: []string{
+						"--endpoint=$(CSI_ENDPOINT)",
+						"--cloud-config=/etc/kubernetes/vcloud/config",
+						"--v=5",
```

Comment:
Can we lower the default verbosity to something reasonable like 2?

Reply:
This has been copied from the addon and kept as is; 5 looks fine to me.